wordpress-mobile/WordPress-Android
# Theme not showing up in My Sites

Fixes #4003 

To test:

1. Log out of the app if already logged in
2. Log in
3. See if Themes list item is shown for a site that the user is an admin in and is a WordPress.com site

Thanks for being a great 🐥 @nbradbury 

Needs review: @tonyr59h
